tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -63,6 +66,699 @@ public function test_wp_enqueue_script() {
 		$this->assertSame( '', get_echo( 'wp_print_scripts' ) );
 	}
 
+	/**
+	 * Test standalone and non standalone inline scripts in the 'after' position of a single main script.
+	 *
+	 * @ticket 12009
+	 */
+	public function test_non_standalone_and_standalone_after_script_combined() {
+		// If a main script containing a `defer` strategy has an `after` inline script, the expected script type is type='javascript', otherwise type='text/template'.
+		unregister_all_script_handles();
```

Comment:
Some of the tests are a little packed together. Can we adding some spacing similar to `test_non_standalone_after_inline_script_with_blocking_main_script()`?